src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -842,5 +767,19 @@ public class SimpleEvents {
 			)
 			.since("INSERT VERSION");
 
+		Skript.registerEvent("Vehicle Move", SimpleEvent.class, VehicleMoveEvent.class, "vehicle move")
+			.description("Called when a vehicle moves.")
+			.examples(
+				"on vehicle move:",
+					"\tbroadcast past event-location",
+					"\tbroadcast event-location"
+			)
+			.since("INSERT VERSION");
+
+		Skript.registerEvent("Vehicle Update", SimpleEvent.class, VehicleUpdateEvent.class, "vehicle update")
+			.description("Called when a vehicle updates.")
```

Comment:
example needed

Reply:
there is an example here?

Reply:
He means an example on how/why it gets updated in the desc. Which idk, I'll have to look

Reply:
ahh i see

Reply:
Honestly cant find anything, even did testing, with a broadcast to see if I could do something to get it trigger. But nothing. Might just have to remove it if I cant figure out how it activates

Reply:
So I asked in  Spigot Discord, MD says it only works for boats  and minecarts, and there's really no "trigger" it gets called every tick if a boat or minecart exists.
Keep or Remove?

Reply:
get it outta here

Reply:
Okie